libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/VISIONTabsView.tsx

```diff
@@ -0,0 +1,337 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { DefaultEffects, PivotItem, Stack, Text } from "@fluentui/react";
+import { CausalInsightsTab } from "@responsible-ai/causality";
+import {
+  WeightVectorOption,
+  ModelTypes,
+  WeightVectors,
+  ModelAssessmentContext,
+  defaultModelAssessmentContext,
+  IModelAssessmentContext
+} from "@responsible-ai/core-ui";
+import { CounterfactualsTab } from "@responsible-ai/counterfactuals";
+//import { DatasetExplorerTab } from "@responsible-ai/dataset-explorer";
+import { VISIONDatasetExplorerTab } from "../../../../../../dataset-explorer/src/lib/VISIONDatasetExplorerTab"
```

Comment:
to reference files from other package, we need to reference by package name instead of relative path.